Annex60/BoundaryConditions/WeatherData/Examples/ReaderTMY3HDirNor.mo

```diff
@@ -0,0 +1,58 @@
+within Annex60.BoundaryConditions.WeatherData.Examples;
```

Comment:
This model should be in a `Validation` package as it validates a configuration of a model.